Lms.Data/Repositories/CourseRepository.cs

```diff
@@ -99,7 +99,16 @@ public async Task<IEnumerable<Course>> GetAllCourses()
 
     public async Task<Course> GetCourseById(int id)
```

Comment:
returns course with id by course id I assume. Name is different from other methods in same file